applications: Fixed displaying QR code link for Matter bridge (#14675)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

The issues in this section are related to the :ref:`ug_matter` protocol.

.. rst-class:: v2-6-0

> KRKNWK-18769: Bridge application does not print hyperlink for displaying the setup QR code in logs.
>   This happens, because the log module that displays this log was disabled.

Reviewer: ```suggestion
KRKNWK-18769: :ref:`matter_bridge_app` application does not print the hyperlink for displaying the setup QR code in the log.
  This happens because the log module that displays this log entry has been disabled.
```

  **Workaround:** Remove the following line from the :file:`src/chip_project_config.h` header file:

  .. code-block:: C

>     #define CHIP_CONFIG_LOG_MODULE_AppServer_PROGRESS 0

Reviewer: ```suggestion
     #define CHIP_CONFIG_LOG_MODULE_AppServer_PROGRESS 0
```
(add one space)
